blocks/library/paragraph/index.js

```diff
@@ -91,10 +93,38 @@ class ParagraphBlock extends Component {
 		this.nodeRef = node;
 	}
 
-	setFontSize( fontSize ) {
-		const { setAttributes } = this.props;
+	getFontSize() {
+		const { customFontSize, fontSize } = this.props.attributes;
+		if ( fontSize ) {
+			const fontSizeValue = FONT_SIZES[ fontSize ];
+			if ( fontSizeValue ) {
+				return fontSizeValue;
+			}
+			const fontSizeNumeric = parseInt( fontSize ); //compatibility with old font size attribute.
```

Comment:
Not certain if this back compatibility logic should exist. But without it, although the existing blocks are not changed, in the front-end or markup. The editor does not show existing blocks with custom sizes with the correct size (but does not changes them so they continue to be ok in the frontEnd).

Reply:
We can keep it for a few releases.

Reply:
I'm fine with not doing this conversion and just dropping the logic.

Reply:
Sorry @mtias I was in the middle of adding the conversion logic and did not check your message.
The conversion logic seems to be working fine now and correctly converts exist fontSize to customFontSize, using our deprecation mechanism, as the type is different before it was number now it is string I think having this logic is fine and does not cause problems with the new paragraph blocks,.
I did some tests and it looks like things are working well.